packages/web-app/src/utils/types.ts

```diff
@@ -244,6 +245,13 @@ export type ActionAddMember = {
   }
 };
 
+export type ActionCreateGroup = {
+  name: 'create_group';
+  inputs: {
+    groupName: string;
```

Comment:
We also need address[]

Reply:
We are using address input from the add_member card.

Initially two actions are created: (to add at least one member)
`[
  { name: 'create_group', inputs: { groupName: ... } },
  { name: 'add_member', inputs: { address: ... },
]`

As more addresses are added to the form, more actions with the name 'add_member' are added to the array.

Reply:
That could work for the frontend and like the way to present it, but won't work to send the action to the contract. The create group function needs to parameters name and addresses, we need both parameters in the encoder to get the action encoded an add it to the proposal to then be persistent on chain

Reply:
Oh, ok. I understood. So, i need to do some refactor on the 'create_group' and 'add_member' flows.

Reply:
Not sure how this will work, still don't see addresses[] in the action. Take a look on how actions are encoded here on proposal creation https://github.com/Libree/frontend/blob/d7e05a3a50994b87fa3aba0a2213147a58f9ef64/packages/web-app/src/context/createProposal.tsx#L146

There is a for over the action array and all the inputs are needed to encode de action and send the TX. How can be the addresses accessed here?